README.md

```diff
@@ -16,3 +17,12 @@ page](https://docs.github.com/en/get-started/writing-on-github/working-with-adva
 might help with the notation for mathematical expressions.
 
 $f(n)\in o(g(n)) \iff \forall c>0, \exists n_0, \forall n\ge n_0: f(n) < c g(n)$
+
+Definition of $O$: $f(n)\in o(g(n)) \iff \forall c>0, \exists n_0, \forall n\ge n_0: f(n) < c g(n)$
+
+
+Ok, im starting over:
+
+We know that there is a point $n_0$ beyond that's always $f(n)< c* g(n)$ for any positive constant $c$  in the definition of $f(n)\in o(g(n))$. We can then pick a positive constant $a$ that is less then our
```

Comment:
How can $a$ be less than any constant $c$?

Reply:
Your picking a constant $a$ that is greater then 0 but less then constant $c$

Reply:
Why does it have to be less than $c$?

Reply:
That's how it ensures the condition for $f(n)) \in O(g(n))$ is satisfied for all $n\geq n_0$.

Reply:
It's the definitions of Big O notation and little o notation. Big O is saying that for some constant $c$ and all large $n$, $T(n)$ is less then or equal to $c$ times $f(n)$ So $T(n)$ is upper-bounded by $c$.

Reply:
I thought that with the Definition of Big O notation it only requires that there exists some constant $a$ such that $f(n) < a*g(n)$ for all $n \ge n_0$. Not all $a$ but at least one $a$. So even if $a$ is larger than $c$ you can still find a constant $a$ that could be the same as $c$ or less then $c$ that will satisfy the definition of Big O notation.

Reply:
But what about the definition of $o$? That allows any constant.